configure.ac

```diff
@@ -194,4 +208,4 @@ AC_MSG_NOTICE([Summary of build options:
   Library types:      Shared=${enable_shared}, Static=${enable_static}
   LDAP:               ${with_ldap}
   Challenge-Response: ${with_cr}
-])
+])
```

Comment:
I'm not sure what modification you want here because for me this file have completely changed after last commit.

Could you please give me more precisions ?